wiki/deployment/bundling-a-jre.md

```diff
@@ -8,18 +8,37 @@ The solution is to bundle a JRE with your app. This way you know exactly what us
 ## Packaging
 There are a number of tools available for bundling a JRE:
 
-### [jpackage](https://docs.oracle.com/en/java/javase/14/jpackage/packaging-overview.html#GUID-C1027043-587D-418D-8188-EF8F44A4C06A)
+### [Construo](https://github.com/fourlastor-alexandria/construo?tab=readme-ov-file#construo)
+The modern way to minimize and package your libGDX apps for deployment on Windows, Linux, and Mac. Simply call the corresponding gradle command for the target platform. These commands can be called from any OS:
+
+**Mac M1** lwjgl3:packageMacM1<br>
+**Mac OSX** lwjgl3:packageMacX64<br>
+**Linux** lwjgl3:packageLinuxX64<br>
+**Windows** lwjgl3:packageWinX64<br>
+
+This creates a zip file in `lwjgl3/build/construo/dist` containing your game and the minimized JRE for the target platform. See [this section](https://www.youtube.com/watch?v=VF6N_X_oWr0&t=1088s) in the GDX-Liftoff video.
 
-Jpackage is a modern solution for providing native packaging options on Windows, MacOS and Linux introduced with [JEP-343](https://openjdk.java.net/jeps/343). It can be used to create an EXE that starts your bundled application via an embedded JRE.
+### [Graal Native Image](https://www.graalvm.org/latest/reference-manual/native-image/)
+Graal Native Image is a way to compile Java code ahead-of-time to create a native executable. This differs from other techniques requiring you to bundle a Java JRE with your game. Native executables are much smaller and require less resources to run, starting up almost instantaneously. To enable it you must set the following in your gradle.properties file:
+
+```
+enableGraalNative=true
+```
 
-See [this guide](https://github.com/raeleus/skin-composer/wiki/libGDX-and-JPackage) for more information on how to use it. A video version can be found [here](https://www.youtube.com/watch?v=R7CMXeQ11GM).
+You must also have a Graal JDK installed. Please note that there is a higher burden regarding reflection/resousource use and it is not expected to work out of the box. For more information take a look at the [official docs](https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html)
```

Comment:
Hm, what is a "resousource" and where can I obtain one?

Reply:
Good catch. Thank you.